### ISSUE TITLE
"Guide Options" - Context Responses for changing radio buttons

### ISSUE THREAD

**Issue by Matthew-Moore240** [phetsims/ratio-and-proportion#203]
**No Tick Marks**
Customize play area without tick marks.
Explore play area without tick marks.
No tick marks in play area.
Customize play area with only words

**Tick Marks** 
Customize play area with tick marks
Explore play area with tick marks.
Tick marks activated in play area.
Customize play area with words and some numbers.

**Numbered Tick Marks**
Customize play area with numbered tick marks.
Explore play area with numbered tick marks.
Numbered tick marks activated in play area.
Customize play area with mostly numbers and some words.

**Comment by Matthew-Moore240** [comment]
**No Tick Marks**
Explore play area without numbers

**Tick Marks**
Explore play area with some numbers.
Explore play area with ordinal numbers.
Explore play area with minimal numbers.

**Numbered Tick Marks**
Explore play area with mostly numbers.
Explore play area with numbers.

@emily-phet Still brainstorming a way to be slightly more specific on how to cue tick marks and numbered tick marks as different

**Comment by terracoda** [comment]
@Matthew-Moore240, I understand your direction here, now, apologies for my confusion.

The style of these phrases is more like "help text" than responsive text. A response in this context often describes what has changed. In this case what has changed is the set-up or described experience in the Play Area.

The goal of the response would be to communicate what's important about this change to the Play Area. What is important about choosing "Tick Marks", or "Numbered Tick Marks" or "No Tick Marks." The radio buttons themselves have good labels, so I think they are good.

You/we could consider adjusting the help text. It is currently very general.
- "Choose how you see or hear your guided exploration."

For example, we could more explicitly talk about position of the hands and the Play Area.
- "Choose how hand position is described as you move them up and down."
- "Choose how hand position is described as you move them."

Possible responses:
- Tick Marks 
    - Evenly spaces horizontal lines added to Play Area.
    - Evenly spaces horizontal lines guide the hands.
- Numbered Tick Marks 
    - Numbered horizontal limes added to Play Area.
    - Numbered horizontal lines guide the hands.
- No Tick Marks 
    - No visual guides in Play Area, general regions described.

I find "No Tick Marks" the hardest.

**Comment by terracoda** [comment]
Another help text option:
- Choose type of guidance when moving the hands.

**Comment by emily-phet** [comment]
Just a quick thought - we might want to be careful about taking the terms "guide" and "guidance" too far. The visual display of the lines is a visual guide to the eye, but the descriptions you hear with these on are not quite that (in my opinion). They are additional useful information. "guide" and "guidance" could be interpreted in more of an active way than we mean, there is no active guidance provided by these features. I wouldn't want someone to be disappointed or confused because they thought more "guidance" (as in, more active help in some way) was going to be provided through selection of these options.

**Comment by terracoda** [comment]
@emily-phet, I agree, but what words besides "lines", "tick marks", "guides" can we use? We have accoustic tick marks in enhanced sounds, so there are accoustic guides in addition tot he described tick marks.

I am finding the different modalities challenging :-)

**Comment by terracoda** [comment]
How about these responses?
- Tick Marks 
    - Move hands in relation to evenly spaced tick marks.
- Numbered Tick Marks 
    - Move hands in relation to numbered tick marks.
- No Tick Marks 
    - Move hands with no tick marks.

**Comment by terracoda** [comment]
Or maybe?
No Tick Marks
- Move hands in relation to general position in Play Area.

**Comment by Matthew-Moore240** [comment]
I am totally not opposed to doing it the way @terracoda is describing but I do think we benefit more from them sounding a bit more help text like. At this point if we can give the learner clear and concise help that is in line with the visual more and doubles as a way to clear up some of the more confusing aspects of the sim I would rather go with that first and then tune according to the interviews.

**Comment by terracoda** [comment]
This is a simple interaction, so let's not over think it. That said, I am quite stumped for the help text. I think the responses for a particular setting is more straight forward.

I am also trying to think about how the help text would work in a Self-Voicing experience.
 
How about any of these?
- Set up different marks for the Play Area as needed.
- Choose different marks for the Play Area as needed.
- Choose between general and specific markings for the Play Area as needed.
- Switch between general and specific markings for Play Area as needed.

What we currently is:
- Choose how you see or hear your guided exploration.

**Comment by terracoda** [comment]
Alternatively, let's just simplify the situation and change the heading to "Tick Mark Options", and have no help text. The labels and responses can clearly communicate  the changes:

**Tick Mark Options**
- No Tick Marks - response upon selecting: Tick Marks hidden.
- Tick Marks - response upon selecting: Evenly-spaced horizontal lines added to Play Area.
- Numbered Tick Marks -  response upon selecting: Evenly-spaced numbered lines added to Play Area.

This wording does align with what we say in the screen summary, and it's pretty clear and simple.

@Matthew-Moore240 and @emily-phet, what do you think?

**Comment by terracoda** [comment]
@BLFiedler you are also welcome to chime in about changes to "Guide Options."

**Comment by Matthew-Moore240** [comment]
I like this idea 

> 
> 
> Alternatively, let's just simplify the situation and change the heading to "Tick Mark Options", and have no help text. The labels and responses can clearly communicate the changes:
> 
> **Tick Mark Options**
> 
>     * No Tick Marks - response upon selecting: Tick Marks hidden.
> 
>     * Tick Marks - response upon selecting: Evenly-spaced horizontal lines added to Play Area.
> 
>     * Numbered Tick Marks -  response upon selecting: Evenly-spaced numbered lines added to Play Area.
> 
> 
> This wording does align with what we say in the screen summary, and it's pretty clear and simple.
> 
> @Matthew-Moore240 and @emily-phet, what do you think?

**Comment by terracoda** [comment]
Awesome, that's 2 votes, one from me and one from you. I am assigning over to @zepumph.

**Comment by terracoda** [comment]
@zepumph, to summarize:
- Change heading for radio group to: "Tick Mark Options".
- Remove help text.
- Implement new context responses.

**Comment by zepumph** [comment]
Easy stuff done, see you over in https://github.com/phetsims/ratio-and-proportion/issues/203#issuecomment-710600010

**Comment by zepumph** [comment]
Context responses implemented above. 

>"Tick Marks hidden"

I wanted to use this same string as what is already in the disabled combo box, so I made that one lowercase to match this usage:
![image](https://user-images.githubusercontent.com/6856943/96325030-41b69b80-0fd1-11eb-9cff-2587f8d92aff.png)


Also, should "Evenly-spaced horizontal" (or numbered) have a comma in between the two adjectives?

**Comment by terracoda** [comment]
@zepumph, that makes perfect sense.
The new heading and responses look good to me. Closing.

**Comment by terracoda** [comment]
Oops, re-opening, missing periods in two responses.

**Comment by terracoda** [comment]
@zepumph, I added the periods in c7ad9d4.
Whoo hoo, an actual commit.
